logprep/runner.py

```diff
@@ -230,6 +249,15 @@ def reload_configuration(self):
                 + str(error)
```

Comment:
I am not sure if you have to add the original error message here, as the verification of the configuration already logs the errors as a batch. The only down side that might could use some refactoring is that the error messages from the verification are not logged but just printed in red.

Reply:
I would leave it as is, because I didn't wrote it and didn't changed it